config/default-disco.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
       mediaSrc: [staticHost],
     },
   },
-  taarParamsToUse: [
+  discoParamsToUse: [
```

Comment:
The issue said this could use more comments; I agree. I think here adding:

```
// Firefox sets these query params when loading the Discovery Pane.
// These are the allowed list of query params we will forward to AMO for statistics.
discoParamsToUse: [
...
```

would be great.